cunumeric/deferred.py

```diff
@@ -58,9 +57,19 @@
     import numpy.typing as npt
 
     from legate.core import FieldID, Region
+    from legate.core.operation import AutoTask, ManualTask
 
+    from .config import FFTDirection, WindowOpCode, _FFTType
```

Comment:
Possibly we should make `_FFTType` public, if it's useful for typing

Reply:
Seems reasonable to me, also maybe there is a better place for it? `.config` is a bit of a jumble presently

Reply:
I changed it to public for now 3066af7b24802464b88b03bfb2320f7dc91782c1